build/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   glob: ^1.1.0
 
 dev_dependencies:
-  build_barback: ^0.2.0
+  build_barback: ^0.4.0
```

Comment:
No it's not, build_barback has a public implementation of the Resolver interface which saw breaking changes.